### ISSUE TITLE
[BUG REPORT] - Title

### ISSUE THREAD

**Issue by Albin323** [Maximus7474/lb-phone-simcards#4]
**Describe the bug**
You can only use the simcard system once. Then this error comes up.

**To Reproduce**
1. Go to 'inventry'
2. Click on 'Simcard, use'
3. Scroll down to 'Error show in server console'

**Expected behavior**
You should be able to use it whenever, or atlesat on a cooldown to make rp more intresting. Also, if possible give a simcard with the number already on the phone. So it's like you are changing it.

**Screenshots**
![image](https://github.com/user-attachments/assets/29f93adb-d844-4b38-a9a0-2deaba868870)

**Additional context**

**Comment by Albin323** [comment]
The cause

it doesn't get an ID in the database.

something is wrong here

function Utils.GenerateSerialNumber(length)
    local charset = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890'
    if type(length) ~= 'number' then length = 6 end

    local serial = ''
    for i = 1, length do
        serial = serial .. charset:sub(math.random(1, #charset), 1)
    end

    return serial
end

**Comment by Albin323** [comment]
Code:

function Utils.GenerateSerialNumber(length)
    local charset = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890'
    if type(length) ~= 'number' then length = 6 end

    local serial = ''
    for i = 1, length do
        serial = serial .. tostring(math.random(1, #charset), 1)
    end

    return serial
end

makes it atleast give "random" id's... But it's way more than 6 characters.

![image](https://github.com/user-attachments/assets/dca627f4-eb1a-4892-b5df-ea9845a4b628)
After a few test (on other things aswell. :) )

Also, no matter what the sim card gets removed.

**Comment by Maximus7474** [comment]
Fixed in [9c8b029](https://github.com/Maximus7474/lb-phone-simcards/commit/9c8b029624d8fa49aa28046668e04b9184d630e5)

The reason your code doesn't generate the wanted length is because of this: 
```lua
tostring(math.random(1, #charset), 1)
```
It should only take one parameter not two. It would of been sufficient to just have:
```lua
serial = serial .. math.random(1, #charset)
```
